lib/holdings.rb

```diff
@@ -59,8 +59,9 @@ def mhld
     end
   end
 
-  def as_json(*)
```

Comment:
so the * here has always been superfluous?  No other consumers besides requests app?